content/faq/kube-lego-privacy-error.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Error: Your connection is not private"
+description: "If your getting a TLS error for a service leveraging `kube-lego`, then try deleting the TLS secret to let `kube-lego` generate a new one."
+tags:
+- kube-lego
+- nginx-ingress
+- tls
+- kubernetes
+- letsencrypt
+---
+
+# Question
+
+We're using `kube-lego` together with the standard nginx ingress controller for kubernetes. The site was working fine with TLS, but after we changed the hostname and redeployed, we started getting the following error.
+
+{{< img src="/assets/kube-lego-privacy-error-9d1bee77.png" title="TLS Privacy Error" >}}
+
+# Answer
```

Comment:
I guess this error can also happen when
* DNS resolve broken (errors with external DNS)
* Achieve  let's encrypt limits

Reply:
Updated